src/components/AccountCreation.jsx

```diff
@@ -0,0 +1,185 @@
+import { useState, useRef } from 'react';
+import { Form, Button, Card, Alert } from 'react-bootstrap';
+import FilterDropdown from './common/FilterDropdown';
+
+function AccountCreation() {
+  // @TODO Fetch the schools from the database
+  // For now mock the data
+  const schools = [
+    'Tufts University',
+    'Ohio State University',
+    'Tower Hill School',
+    'Wesleyan University'
+  ];
```

Comment:
fine for now, but just know that this might look like something like

```ts
const schools = [
  {
    id: "1234-1234-1234-1234",
    name: "Tufts University",
    slug: "tufts",
    domain: "tufts.edu",
    ...
  },
  ...
]
```